Description: Freeze so number at 19
 .
 gsl (2.4+dfsg-2) unstable; urgency=medium
 .
   * configure.ac: Once again reset sonumber to 19 (Closes: #869778, #869695)
Author: Dirk Eddelbuettel <edd@debian.org>
Bug-Debian: https://bugs.debian.org/869695
Bug-Debian: https://bugs.debian.org/869778

--- gsl-2.4+dfsg.orig/configure.ac
+++ gsl-2.4+dfsg/configure.ac
@@ -60,7 +60,7 @@ dnl See https://www.gnu.org/software/lib
 dnl for more detailed info
 
 dnl
-GSL_CURRENT=23
+GSL_CURRENT=19
 GSL_REVISION=0
 GSL_AGE=0
 dnl
